src/pages/AdminAdvancedSettings/AdminAdvancedSettings.tsx

```diff
@@ -0,0 +1,94 @@
+import React, { useCallback, useMemo, useState } from "react";
+import { Nav } from "react-bootstrap";
+import classNames from "classnames";
+
+import { useVenueId } from "hooks/useVenueId";
+import { useConnectCurrentVenueNG } from "hooks/useConnectCurrentVenueNG";
+import { useUser } from "hooks/useUser";
+import { useRoles } from "hooks/useRoles";
+import { useIsAdminUser } from "hooks/roles";
+
+import { Venue_v2 } from "types/venues";
```

Comment:
```suggestion
import { Venue_v2 } from "types/venues";

import { useVenueId } from "hooks/useVenueId";
import { useConnectCurrentVenueNG } from "hooks/useConnectCurrentVenueNG";
import { useUser } from "hooks/useUser";
import { useRoles } from "hooks/useRoles";
import { useIsAdminUser } from "hooks/roles";

```

Reply:
Fixed in: https://github.com/sparkletown/sparkle/pull/1511/commits/484c4afb77a3b370f73be7ab7dbd7911b30553c0